osu.Game.Tests/Visual/Online/TestSceneWikiMarkdownContainer.cs

```diff
@@ -142,7 +142,7 @@ public void TestBlockImage()
         {
             AddStep("Add paragraph with block image", () =>
             {
-                markdownContainer.CurrentPath = "https://dev.ppy.sh/wiki/Interface/";
+                markdownContainer.CurrentPath = "https://dev.ppy.sh/wiki/images/Client/Interface/";
```

Comment:
I don't get this change. It's not going to match what the `CurrentPath` is set to when visiting the "Interface" page. It shouldn't need to change?

Reply:
The reason I've changed these links is because the image links used in the visual tests are broken and do not redirect, so I have updated them to point to working links.

Old link that 404s: https://dev.ppy.sh/wiki/Interface/img/intro-screen.jpg
New link that works: https://dev.ppy.sh/wiki/images/Client/Interface/img/intro-screen.jpg

This is a test only change to get the images to display.

Reply:
I also think this change looks pretty wrong.

This test is a bit jury-rigged, but setting the current path of a markdown document to an image-related URL just doesn't reflect the reality of how the thing is used in actuality. I would have much preferred to leave the `CurrentPath` as it was everywhere and just fix the image URLs inside the markdown documents themselves.

Reply:
Reverted changes to `CurrentPath` and have only changed the markdown links